opm/parser/share/keywords/000_Eclipse100/W/WSEGITER

```diff
@@ -0,0 +1,6 @@
+     {"name" : "WSEGITER", "sections" : ["SPECIAL", "SCHEDULE"], "size" : 1, "items" : [
```

Comment:
The `SPECIAL` section should be removed, the `SPECIAL` section was a mistake which was introduced at some time, and have appeared again and again after.